libs/src/api/track.js

```diff
@@ -108,6 +107,22 @@ class Track extends Base {
     return this.creatorNode.getUnlistedTracks()
   }
 
+  /**
```

Comment:
Just putting a note since since @sddioulde  & I chatted about this -- the corresp. client code doesn't use this libs bindings (and uses API client), but he had already added them, so im good shipping for posterity/completeness!